libs/components/tiles/src/lib/modules/tiles/tile-dashboard/fixtures/tile-context.fixture.ts

```diff
@@ -1,3 +1,3 @@
 export class TileTestContext {
-  public id: string;
+  public id?: string;
```

Comment:
This is a class, so it should be `public id: string | undefined`.

Reply:
Fixed